src/lib/input/index.ts

```diff
@@ -7,50 +7,36 @@
  */
 
 import {NgModule} from '@angular/core';
-import {
-  MdErrorDirective,
-  MdHint,
-  MdInputContainer,
-  MdInputDirective,
-  MdPlaceholder,
-  MdPrefix,
-  MdSuffix
-} from './input-container';
+import {MdInput} from './input';
 import {MdTextareaAutosize} from './autosize';
 import {CommonModule} from '@angular/common';
 import {PlatformModule} from '../core/platform/index';
+import {MdFormFieldModule} from '../form-field/index';
 
 
 @NgModule({
   declarations: [
-    MdErrorDirective,
-    MdHint,
-    MdInputContainer,
-    MdInputDirective,
-    MdPlaceholder,
-    MdPrefix,
-    MdSuffix,
+    MdInput,
     MdTextareaAutosize,
   ],
   imports: [
     CommonModule,
+    MdFormFieldModule,
     PlatformModule,
   ],
   exports: [
-    MdErrorDirective,
-    MdHint,
-    MdInputContainer,
-    MdInputDirective,
-    MdPlaceholder,
-    MdPrefix,
-    MdSuffix,
+    // TODO(mmalerba): We import and re-export the form field module since all existing users of
+    // `MdInput` will need this to continue using `md-input-container`. We may want to keep this
+    // long term since the `MdInput` directive will almost always be used with `md-form-field`.
```

Comment:
Yeah, long-term I think the fact that `form-field` is its own thing is an implementation detail (similar to mdLine) that the users don't need to care about